developing-without-a-build/es-dev-server.md

```diff
@@ -0,0 +1,195 @@
+# Developing Without a Build: es-dev-server
+This article is part of a series on developing without a build:
+- Introduction
+- es-dev-server (this article)
+- Testing (coming soon!)
+
+In the introduction, we looked into different workflows and browser APIs. In this article, we will look into how we can set up `es-dev-server`, and how it helps with development without a build.
+
+## es-dev-server
+
+[es-dev-server](https://www.npmjs.com/package/es-dev-server) is a composable web server that focuses on developer productivity when developing with a build. Through options, you can opt into features such as caching, reloading on file changes, SPA routing, resolving bare module imports and compatibility modes to support older browsers.
+
+## Setup
+To start let's create an empty npm project and install `es-dev-server`:
+
+```bash
+npm init
+cd <your-project-name>
+npm i -D es-dev-server
+```
+
+Create an `index.html` in the root of your project:
+```html
+<!DOCTYPE html>
+<html>
+<head></head>
+
+<body>
+  <div id="app"></div>
+  <script type="module" src="./src/app.js"></script>
+</body>
+
+</html>
+```
+
+Create a `src/app.js` file:
+```javascript
+console.log('hello world');
+```
+
+We can now view our application with `es-dev-server`:
+```bash
+npx es-dev-server --open
```

Comment:
if installed in the step above, why use npx here?

Reply:
`npx` executes it from the local node modules folder, right? without it it will try to execute it from globa.